includes/Modules/Sign_In_With_Google.php

```diff
@@ -126,12 +126,11 @@ function () {
 
 		add_action( 'admin_action_' . self::ACTION_DISCONNECT, fn () => $this->handle_disconnect_user() );
 
-		add_action( 'login_form', $this->get_method_proxy( 'render_signin_button' ) );
-
 		add_action( 'show_user_profile', $this->get_method_proxy( 'render_disconnect_profile' ) ); // This action shows the disconnect section on the users own profile page.
 		add_action( 'edit_user_profile', $this->get_method_proxy( 'render_disconnect_profile' ) ); // This action shows the disconnect section on other users profile page to allow admins to disconnect others.
 
-		add_action( 'woocommerce_login_form_start', $this->get_method_proxy( 'render_signin_button' ) );
+		// (Potentially) render the Sign in with Google button on all pages.
+		add_action( 'wp_footer', $this->get_method_proxy( 'render_signin_button' ) );
```

Comment:
Actually, i am not sure that this is a good approach to use this version of the sign-in button script on all pages, especially on those that don't need the sign-in button. We should better revert changes in this function and create a new method that renders just one-tap script.

Plus, there is no `wp_footer` action call on the login page because it uses `login_footer` instead: https://github.com/WordPress/WordPress/blob/master/wp-login.php#L448

Reply:
Oh, shoot, good point.

Yeah, I wondered about rendering only the script, but knowing in the future we want to actually be _able_ to render the sign-in buttons on any page _and_ the code already checks if it's on the login pages, maybe it'd be better for now to restrict the button transformations to login pages but always render the script?

We could change the method name to something like `load_siwg_script` or something instead 🤔

Reply:
The code already only does button replacements on the relevant pages: https://github.com/google/site-kit-wp/blob/7ac68182bb0985028164f9b350b18efee040ad50/includes/Modules/Sign_In_With_Google.php#L346-L352

I think that's fair for now, given in the future we'll want to allow the buttons to replaced on all pages, so rather than separate them out only to combine them again later, we can leave them as-is.

Reply:
Ok, but let's not define a new div and render the button if it is neither the login page nor a woocommerce login form. Also, there is no sense in adding the redirect_to part as well because one-tap will never have that parameter on random pages.

Reply:
I mean that we need to skip the following blocks if it is not a login page/form:

https://github.com/google/site-kit-wp/blob/63f3394a3a30255e64e9cb50d314a3552f6cbaa8/includes/Modules/Sign_In_With_Google.php#L330-L335
https://github.com/google/site-kit-wp/blob/63f3394a3a30255e64e9cb50d314a3552f6cbaa8/includes/Modules/Sign_In_With_Google.php#L358
https://github.com/google/site-kit-wp/blob/63f3394a3a30255e64e9cb50d314a3552f6cbaa8/includes/Modules/Sign_In_With_Google.php#L364-L368

Reply:
That makes sense 👍🏻 

I've made the change 🙂